requirements.in

```diff
@@ -1,35 +1,35 @@
 # The file contains the direct ckan requirements.
 # Use pip-compile to create a requirements.txt file from this
 Babel==2.3.4
-Jinja2==2.8
-Pylons==0.9.7
 Beaker==1.7.0 # need to pin this because of https://github.com/bbangert/beaker/commit/fc511ceda4305fcf54919b3f081fed7790e2fef3
```

Comment:
Beaker should be also safe to upgrade.

The comment should be something like `# Needs to be pinned to a more up to date version than the Pylons one`